units/commit_queue.go

```diff
@@ -608,3 +599,45 @@ func setDefaultNotification(username string) error {
 
 	return nil
 }
+
+func updateGithashes(ctx context.Context, githubToken string, projectRef *model.ProjectRef, project *model.Project, patchDoc *patch.Patch) error {
```

Comment:
should there be more tests of this, or should this be unified somehow with what the CLI does?

Reply:
I added a test.
I'm not sure about unifying with the CLI: the CLI can't know the most up-to-date githash in the case where the item is enqueued and something else is merged in the meantime.